### PR TITLE
Update full.sh with e function for all the tasks to get the average  

### PR DESCRIPTION
**Pull request by Imaanpreet**

### CODE REVIEW

**Comment by jhutar** [comment]
Hello. Logs `12-repo-sync-*.log` do not have info to be processed by `reg-average.sh` - please try yourself:
```
wget https://workdir-exporter-.../workspace/ContPerf67/run-2020-03-06T16:53:48+00:00/12-repo-sync-rhel6.log
./reg-average.sh RepoSyncRhel7 12-repo-sync-rhel6.log
```
Sync time here is a duration of the whole command as you are using it in your Kibana dashboard already.

**Comment by jhutar** [comment]
Same for `13-capsule-sync-$capsule_id.log`. No need to investigate via `reg-average.sh`, just use result this command is recording itself.

**Comment by jhutar** [comment]
Same for others. Only place where it makes sense are these Puppet logs. There you would go with these terms:

* `RegisterPuppet*` -> `RegisterPuppet`
* `SetupPuppet*` -> `SetupPuppet`
* `PickupPuppet*` -> `PickupPuppet`

**Comment by jhutar** [comment]
Note that on the puppet logs, the output will be empty if you will try on logs from latest Jenkins run as the test did not worked.

**Comment by Imaanpreet** [comment]
I have updated the full.sh file with the requested changes.

File: experiment/full.sh [hunk 1]
```diff
@@ -67,6 +67,7 @@ h 10-reposet-enable-rhel6.log  "repository-set enable --organization '$do' --pro
 h 10-reposet-enable-rhel7optional.log "repository-set enable --organization '$do' --product 'Red Hat Enterprise Linux Server' --name 'Red Hat Enterprise Linux 7 Server - Optional (RPMs)' --releasever '7Server' --basearch 'x86_64'"
 h 11-repo-immediate-rhel7.log "repository update --organization '$do' --product 'Red Hat Enterprise Linux Server' --name 'Red Hat Enterprise Linux 7 Server RPMs x86_64 7Server' --download-policy 'immediate'"
 h 12-repo-sync-rhel7.log "repository synchronize --organization '$do' --product 'Red Hat Enterprise Linux Server' --name 'Red Hat Enterprise Linux 7 Server RPMs x86_64 7Server'"
+e RepoSyncRhel7 $logs/12-repo-sync-rhel7.log
```
Comment (jhutar) [inline experiment/full.sh]: Please also remove this line.

File: experiment/full.sh [hunk 2]
```diff
@@ -95,15 +96,19 @@ else
     h 20-cv-create-all.log "content-view create --organization '$do' --product 'Red Hat Enterprise Linux Server' --repositories 'Red Hat Enterprise Linux 7 Server RPMs x86_64 7Server','Red Hat Enterprise Linux 6 Server RPMs x86_64 6Server','Red Hat Enterprise Linux 7 Server - Optional RPMs x86_64 7Server' --name 'BenchContentView'"
 fi
 h 21-cv-all-publish.log "content-view publish --organization '$do' --name 'BenchContentView'"
+e Publish $logs/21-cv-all-publish.log
```
Comment (jhutar) [inline experiment/full.sh]: Also remove this.

File: experiment/full.sh [hunk 3]
```diff
@@ -95,15 +96,19 @@ else
...
 s $wait_interval
 h 22-le-create-1.log "lifecycle-environment create --organization '$do' --prior 'Library' --name 'BenchLifeEnvAAA'"
 h 22-le-create-2.log "lifecycle-environment create --organization '$do' --prior 'BenchLifeEnvAAA' --name 'BenchLifeEnvBBB'"
 h 22-le-create-3.log "lifecycle-environment create --organization '$do' --prior 'BenchLifeEnvBBB' --name 'BenchLifeEnvCCC'"
 h 23-cv-all-promote-1.log "content-view version promote --organization 'Default Organization' --content-view 'BenchContentView' --to-lifecycle-environment 'Library' --to-lifecycle-environment 'BenchLifeEnvAAA'"
+e PromoteCV1 $logs/23-cv-all-promote-1.log
```
Comment (jhutar) [inline experiment/full.sh]: Also remove this.

File: experiment/full.sh [hunk 4]
```diff
@@ -95,15 +96,19 @@ else
...
 s $wait_interval
 h 23-cv-all-promote-2.log "content-view version promote --organization 'Default Organization' --content-view 'BenchContentView' --to-lifecycle-environment 'BenchLifeEnvAAA' --to-lifecycle-environment 'BenchLifeEnvBBB'"
+e PromoteCV2 $logs/23-cv-all-promote-2.log
```
Comment (jhutar) [inline experiment/full.sh]: Also remove this.

File: experiment/full.sh [hunk 5]
```diff
@@ -95,15 +96,19 @@ else
...
 s $wait_interval
 h 23-cv-all-promote-3.log "content-view version promote --organization 'Default Organization' --content-view 'BenchContentView' --to-lifecycle-environment 'BenchLifeEnvBBB' --to-lifecycle-environment 'BenchLifeEnvCCC'"
+e PromoteCV3 $logs/23-cv-all-promote-3.log
```
Comment (jhutar) [inline experiment/full.sh]: Also remove this.

File: experiment/full.sh [hunk 6]
```diff
@@ -131,8 +136,11 @@ h 10b-reposet-enable-rhel7.log  "repository-set enable --organization '$do' --pr
 h 10b-reposet-enable-rhel6.log  "repository-set enable --organization '$do' --product 'Red Hat Enterprise Linux Server' --name 'Red Hat Enterprise Linux 6 Server (RPMs)' --releasever '6Server' --basearch 'x86_64'"
 h 10b-reposet-enable-rhel7optional.log "repository-set enable --organization '$do' --product 'Red Hat Enterprise Linux Server' --name 'Red Hat Enterprise Linux 7 Server - Optional (RPMs)' --releasever '7Server' --basearch 'x86_64'"
 h 12b-repo-sync-rhel7.log "repository synchronize --organization '$do' --product 'Red Hat Enterprise Linux Server' --name 'Red Hat Enterprise Linux 7 Server RPMs x86_64 7Server'" &
+e RepoSyncRhel7 $logs/12b-repo-sync-rhel7.log
```
Comment (jhutar) [inline experiment/full.sh]: Also remove this.

File: experiment/full.sh [hunk 7]
```diff
@@ -131,8 +136,11 @@ h 10b-reposet-enable-rhel7.log  "repository-set enable --organization '$do' --pr
...
 h 12b-repo-sync-rhel6.log "repository synchronize --organization '$do' --product 'Red Hat Enterprise Linux Server' --name 'Red Hat Enterprise Linux 6 Server RPMs x86_64 6Server'" &
+e RepoSyncRhel6 $logs/12b-repo-sync-rhel6.log
```
Comment (jhutar) [inline experiment/full.sh]: Also remove this.

File: experiment/full.sh [hunk 8]
```diff
@@ -131,8 +136,11 @@ h 10b-reposet-enable-rhel7.log  "repository-set enable --organization '$do' --pr
...
 h 12b-repo-sync-rhel7optional.log "repository synchronize --organization '$do' --product 'Red Hat Enterprise Linux Server' --name 'Red Hat Enterprise Linux 7 Server - Optional RPMs x86_64 7Server'" &
+e RepoSyncRhel7Optional $logs/12b-repo-sync-rhel7optional.log
```
Comment (jhutar) [inline experiment/full.sh]: Also remove this.

**Comment by jhutar** [comment]
Thank you!

**Pull request merged**